cellpack/autopack/BaseGrid.py

```diff
@@ -193,6 +193,8 @@ def __init__(
             # what about collision ?
 
     def setup(self, boundingBox, space):
+        if space == 0:
+            space = 20
```

Comment:
Can you explain the reason for this? Why 20?

Reply:
it was quicker to do that rather that track down the automatic value calculated based on the ingredient size. So the grid would not be huge for the test. So it needs more testing.

Reply:
can you add a "TODO:" comment here? this doesn't seem like a good thing to leave in production code.